src/indexes/text.cc

```diff
@@ -70,15 +80,21 @@ int Text::RespondWithInfo(ValkeyModuleCtx* ctx) const {
 }
 
 bool Text::IsTracked(const InternedStringPtr& key) const {
-  throw std::runtime_error("Text::IsTracked not implemented");
+  return false;
```

Comment:
We will need the mapping for updates. Right now, this works for `Add` operations which is good for the initial implementation